airqo-platform/AirQo-experiments
# Refactor spatial model

WHAT DOES THIS PR DO?
This PR refactors the spatial modelling code used for air quality prediction

WHAT JIRA STORY/TASK IS RELATED TO THIS PR?
N/A

WHAT IS THE LINK TO THE PR BRANCH

HOW DO I TEST OUT THIS PR?
To run this code, you need to have Python>=3.7 installed on your system.

    Git clone this repo and run these commands
    cd spatial-model
    python -m venv env for Windows OR python3 -m venv env for linux/macOS
    env\scripts\activate for Windows OR source env/bin/activate for linux/macOS
    pip install -r requirements.txt
    jupyter notebook
    Open downloading_data.ipynb and run all to download data. A data.p file should be created in the data/raw older
    Open spatial_model_prediction.ipynb and run all to get predictions. The images/plots folder should have some plots
